pkg/security/scc/matcher.go

```diff
@@ -429,3 +436,38 @@ func requiresPreallocatedFSGroup(constraint *kapi.SecurityContextConstraints) bo
 	}
 	return len(constraint.FSGroup.Ranges) == 0
 }
+
+// AssignConstraints implements a template function to share code between security admission and
+// security policy review endpoints
+func AssignConstraints(matcher SCCMatcher, constraints []*kapi.SecurityContextConstraints, namespace string, client clientset.Interface, assigner SCCAssigner) error {
+	constraints = DeduplicateSecurityContextConstraints(constraints)
+	sort.Sort(ByPriority(constraints))
+	errs := []error{}
+	var (
+		providers []kscc.SecurityContextConstraintsProvider
+		ns        *kapi.Namespace
```

Comment:
Explain what these variables are for, it too me a while to figure it out. Only looking at `CreateProvidersFromConstraints` I discovered there purposes. Copying the comments from there is perfectly fine. Btw. `errs` can be moved into this `var` block as well.